templates/project.py

```diff
@@ -212,7 +212,7 @@ def create_storage_logs_bucket(context, api_names_list):
         'type': 'gcp-types/storage-v1:bucketAccessControls',
         'properties': {
             'bucket': bucket_name,
-            'entity': 'group:cloud-storage-analytics@google.com',
+            'entity': 'group-cloud-storage-analytics@google.com',
```

Comment:
are you sure about this?

Reply:
I remember tinkering a while here, and coming to the conclusion that the docs were lying. I might be wrong though.

Reply:
I continue to believe I'm right https://cloud.google.com/storage/docs/access-logs#delivery

Reply:
Sorry I missed your comment weeks ago. I made this change because the request was failing otherwise, and I checked the docs & saw the indication under the JSON API that dash is required. It makes no sense... but this change made it work again on my test deployments.

Reply:
just to follow up -- you did indeed turn out right here :) just got an error saying "invalid value" for the colon-y version